pkg/services/sqlstore/playlist.go

```diff
@@ -1,113 +1,121 @@
 package sqlstore
 
 import (
+	"context"
+
 	"github.com/grafana/grafana/pkg/bus"
 	"github.com/grafana/grafana/pkg/models"
 )
 
-func init() {
-	bus.AddHandler("sql", CreatePlaylist)
-	bus.AddHandler("sql", UpdatePlaylist)
-	bus.AddHandler("sql", DeletePlaylist)
-	bus.AddHandler("sql", SearchPlaylists)
-	bus.AddHandler("sql", GetPlaylist)
-	bus.AddHandler("sql", GetPlaylistItem)
+func (ss *SQLStore) addPlaylistQueryAndCommandHandlers() {
+	bus.AddHandlerCtx("sql", ss.CreatePlaylist)
+	bus.AddHandlerCtx("sql", ss.UpdatePlaylist)
+	bus.AddHandlerCtx("sql", DeletePlaylist)
```

Comment:
Any reason you haven't made `DeletePlaylist` method part of `SQLStore` struct? Seems like you've changed all other handlers. Maybe you can fix this one as well?